docs/developing-locally-docker.rst

```diff
@@ -105,7 +105,7 @@ The most important thing for us here now is ``env_file`` section enlisting ``./.
     │   ├── .django
     │   └── .postgres
     └── .production
-        ├── .caddy
+        ├── .traefik
```

Comment:
This file does not exist, should remove this line.

Reply:
Indeed, I missed that somehow.